.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,9 +6,9 @@
 - [ ] Commit message starts with `[Resolve #issue-number]`.
 - [ ] Added/Updated unit tests.
 - [ ] Added/Updated integration tests (if applicable).
-- [ ] All unit tests (`make test`) are passing.
+- [ ] All unit tests (`poetry run tox`) are passing.
 - [ ] Used the same coding conventions as the rest of the project.
-- [ ] The new code passes pre-commit validations (`pre-commit run --all-files`).
+- [ ] The new code passes pre-commit validations (`poetry pre-commit run --all-files`).
```

Comment:
I think `poetry run pre-commit run --all-files` is the correct command.

Reply:
fixed